plugins/catalog-backend-module-bitbucket-cloud/config.d.ts

```diff
@@ -53,6 +55,11 @@ export interface Config {
                */
               projectKey?: RegExp;
             };
+            /**
+             * (Optional) TaskScheduleDefinition for the discovery.
+             * @visibility backend
+             */
+            schedule?: TaskScheduleDefinition | string;
```

Comment:
Bit confused by the `| string` here tbh, the code does seem to handle it, but assume that it's an object?

Reply:
ah, you are right. I wanted to follow your suggestion to use `TaskScheduleDefinition` directly which I didn't do before as `TaskScheduleDefinition.frequency` etc. support `Duration` as type, but at the config I would support `string` instead of it.

hmm.. maybe I should revert this back 🤔 

What do you think @Rugvip ?

Reply:
Ah yep I see the issue. Perhaps revert then and revisit in a followup. Maybe a `TaskScheduleDefinitionConfig` could be the way to go? Would hate for the config block to be copied all over the place, especially with the existence of `readTaskScheduleDefinition`.

On that note, do you think `readTaskScheduleDefinitionConfig` would be a better name?

Reply:
> On that note, do you think readTaskScheduleDefinitionConfig would be a better name?

makes sense. Or maybe `readTaskScheduleDefinitionFromConfig`

Reply:
> Maybe a TaskScheduleDefinitionConfig could be the way to go?

you mean as a type at backend-task and then imported / only used at the config.d.ts file(s)?

Reply:
I've renamed the function to `readTaskScheduleDefinitionFromConfig` and added the new type `TaskScheduleDefinitionConfig` used then at the config.d.ts